ledger/ledger-api-common/src/test/suite/scala/com/digitalasset/ledger/api/validation/SubmitRequestValidatorTest.scala

```diff
@@ -83,9 +83,8 @@ class CommandSubmissionRequestValidatorTest
     )
   }
 
-  val sut = new CommandSubmissionRequestValidator(
```

Comment:
`sut`, no like? :)

Reply:
![sup](https://media.giphy.com/media/6UszV4EBkkZkQ/giphy.gif)

Reply:
I actually don't like the whole idea of using a command validator to validate values, but one step at a time. :wink: